### ISSUE TITLE
Start up diagnostics aren't human readable 

### ISSUE THREAD

**Issue by andreasohlund** [Particular/NServiceBus#5050]
One of the primary requirements of StartUpDiagnostics is that it is human readable.

This way both customers and us (when we do support tasks) can easily open, read and groc the contents without needing to resort to using extra tooling

At the moment the current output is not readable.

```
{"AssemblyScanning":{"Assemblies":["NServiceBus.Core, Version=7.0.0.0, Culture=neutral, 
PublicKeyToken=9fc386479f8a226c"],"ErrorsThrownDuringScanning":false,"SkippedFiles":[]},
"Transport":{"Type":"NServiceBus.LearningTransport","Version":"7.0.0"},"Recoverability":
{"ImmediateRetries":5,"DelayedRe
tries":3,"DelayedRetriesTimeIncrease":"0:00:10","ErrorQueue":"error","UnrecoverableExcep
tions":["NServiceBus.MessageDeserializationException"]},"HostInformation":{"HostId":"47f
4a845-2c9e-9663-df88-992240c011ca","DisplayName":"MACBOOK","Properties":{"Machine":"MACB
OOK","ProcessID":"984","UserName":"Simon","PathToExecutable":"C:\\Users\\Simon\\AppData\
\Local\\JetBrains\\Installations\\ReSharperPlatformVs15_d7eabe4d\\JetBrains.ReSharper.Ta
skRunner.CLR45.x64.exe"}},"Features":[{"Name":"NServiceBus.HostStartupDiagnostics","Enab
ledByDefault":true,"Active":true,"PrerequisiteStatus":{"IsSatisfied":true,"Reasons":[]},
"Dependencies":[],"Version":"7.0.0","StartupTasks":["WriteStartupDiagnostics"],"Dependen
ciesAreMet":true},{"Name":"NServiceBus.ReceiveStatisticsFeature","EnabledByDefault":true
,"Active":true,"PrerequisiteStatus":{"IsSatisfied":true,"Reasons":[]},"Dependencies":[],
"Version":"7.0.0","StartupTasks":[],"DependenciesAreMet":true},{"Name":"NServiceBus.InMe
moryTransactionalStorageFeature","EnabledByDefault":false,"Active":false,"PrerequisiteSt
atus":null,"Dependencies":[],"Version":"7.0.0","StartupTasks":null,"DependenciesAreMet":
false},{"Name":"NServiceBus.Recoverability","EnabledByDefault":true,"Active":true,"Prere
quisiteStatus":{"IsSatisfied":true,"Reasons":[]},"Dependencies":[["NServiceBus.Features.
RootFeature","NServiceBus.Features.DelayedDeliveryFeature"]],"Version":"7.0.0","StartupT
asks":[],"DependenciesAreMet":true},{"Name":"NServiceBus.StorageInitializer","EnabledByD
efault":true,"Active":true,"PrerequisiteStatus":{"IsSatisfied":true,"Reasons":[]},"Depen
dencies":[],"Version":"7.0.0","StartupTasks":["CallInit"],"DependenciesAreMet":true},{"N
ame":"NServiceBus.SerializationFeature","EnabledByDefault":true,"Active":true,"Prerequis
iteStatus":{"IsSatisfied":true,"Reasons":[]},"Dependencies":[],"Version":"7.0.0","Startu
pTasks":[],"DependenciesAreMet":true},{"Name":"NServiceBus.StaticHeaders","EnabledByDefa
ult":true,"Active":false,"PrerequisiteStatus":{"IsSatisfied":false,"Reasons":["No static
 outgoing headers registered"]},"Dependencies":[],"Version":"7.0.0","StartupTasks":null,
 "DependenciesAreMet":true},{"Name":"NServiceBus.Receiving","EnabledByDefault":true,"Act
 ive":true,"PrerequisiteStatus":{"IsSatisfied":true,"Reasons":[]},"Dependencies":[],"Ver
 sion":"7.0.0","StartupTasks":[],"DependenciesAreMet":true},{"Name":"NServiceBus.Sending
 ","EnabledByDefault":true,"Active":true,"PrerequisiteStatus":{"IsSatisfied":true,"Reaso
 ns":[]},"Dependencies":[],"Version":"7.0.0","StartupTasks":["PrepareForSending"],"Depen
 denciesAreMet":true},{"Name":"NServiceBus.Features.Audit","EnabledByDefault":true,"Acti
 ve":false,"PrerequisiteStatus":{"IsSatisfied":false,"Reasons":["No configured audit que
 ue was found"]},"Dependencies":[],"Version":"7.0.0","StartupTasks":null,"DependenciesAr
 eMet":true},{"Name":"NServiceBus.Features.MessageCausation","EnabledByDefault":true,"Ac
 tive":true,"PrerequisiteStatus":{"IsSatisfied":true,"Reasons":[]},"Dependencies":[],"Ve
 rsion":"7.0.0","StartupTasks":[],"DependenciesAreMet":true},{"Name":"NServiceBus.Featur
 es.MessageCorrelation","EnabledByDefault":true,"Active":true,"PrerequisiteStatus":{"IsS
 atisfied":true,"Reasons":[]},"Dependencies":[],"Version":"7.0.0","StartupTasks":[],"Dep
```

**Comment by andreasohlund** [comment]
Backlogging this for now cc @SimonCropp

**Comment by timbussmann** [comment]
it might be worthwhile mentioning that SimpleJson doesn't support pretty-printing atm (see https://github.com/facebook-csharp-sdk/simple-json/issues/79).

**Comment by mikeminutillo** [comment]
[Released in 7.4](https://github.com/Particular/NServiceBus/releases/tag/7.4.0)
